### PR TITLE
Repository

### PR DESCRIPTION
**Pull request by thaonp104**
## Related Tickets:
- [ 28511](https://edu-redmine.sun-asterisk.vn/issues/28511)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by thaonp104** [comment]
ready

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
     }
 
     public function getdate()
     {
-        $courseUser = auth()->user()->courseUserActive->load('course');
-        $subjectUser = auth()->user()->subjectUsersActive->load('subject');
+        $courseUser = $this->userRepo->getCourseUserActiveByUser(auth()->user())
+            ->load('course');
+        $subjectUser = $this->userRepo->getSubjectUsersActiveByUser(auth()->user())
+            ->load('subject');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/HomeController.php]: load này đưa hết vào repo nhé

File: resources/views/supervisor/manage-task/list-tasks.blade.php [hunk 2]
```diff
@@ -133,11 +133,12 @@ class="table table-theme table-row v-middle dataTable no-footer"
                                                             </tr>
                                                         </thead>
                                                         <tbody>
-                                                            @foreach ($tasks as $index => $task)
+                                                            <?php $index = config('number.init') ?>
```
Comment (huyenttm-0901) [inline resources/views/supervisor/manage-task/list-tasks.blade.php]:
```suggestion
                                                            @php $index = config('number.init') @endphp
```

**Comment by thaonp104** [comment]
ready

**Pull request merged**